src/main/kotlin/ar/edu/unahur/obj2/semillasAlViento/Planta.kt

```diff
@@ -1,39 +1,41 @@
 package ar.edu.unahur.obj2.semillasAlViento
 
-abstract class Planta(val anioObtencionSemilla: Int, var altura: Float) {
+abstract class Planta(val anioObtencionSemilla: Int, var altura: Double) {
```

Comment:
La altura no debía cambiar.
```suggestion
abstract class Planta(val anioObtencionSemilla: Int, val altura: Double) {
```